### PR TITLE
Education Profile

### PR DESCRIPTION
**Pull request by aidanouckama**
- Added implementation for user to add education / school
- Utilized an API from a GitHub repo: https://github.com/Hipo/university-domains-list
- Slightly changed the profile schema to include school
- Created a new search query for education

### CODE REVIEW

**Review by aidanouckama — COMMENTED** [review]
omg this is so good aidan good job

File: client/src/components/SchoolQuerySearch.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+                    className={
+                        'z-20 w-full rounded-full bg-gray-100 px-4 py-2 text-gray-800 outline-none'
+                    }
+                    type="text"
+                    value={query}
+                    onChange={handleInputChange}
+                    placeholder="Harvard University"
+                />
+                <div
+                    className={`z-10 max-h-36 w-full -translate-y-4 overflow-y-scroll bg-white shadow-md ${hasResults ? 'absolute top-full' : ''}`}
```
Comment (Lewin-B) [inline client/src/components/SchoolQuerySearch.js]: The search results are on too low of a Z index it's going under the Experience Form component

File: client/src/components/SchoolQuerySearch.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+    return (
+        <>
+            <form
+                className={`flex w-full flex-col items-center justify-center ${hasResults ? 'relative' : ''}`}
+                onSubmit={(e) => e.preventDefault()} // Prevent form submission
+            >
+                <input
+                    className={
+                        'z-20 w-full rounded-full bg-gray-100 px-4 py-2 text-gray-800 outline-none'
```
Comment (Lewin-B) [inline client/src/components/SchoolQuerySearch.js]: The color of the input is a little bright in my opinion

File: client/src/pages/Profile.js [hunk 3]
```diff
@@ -432,6 +432,18 @@ function Profile() {
                         ) : (
                             <></>
                         )}
+
+                        {/* Education */}
+                        <div className="flex h-full w-full flex-col items-center justify-center gap-5 bg-white p-10 shadow-md">
```
Comment (Lewin-B) [inline client/src/pages/Profile.js]: This whole section should be the same color as everything else on the page

**Review by Lewin-B — CHANGES_REQUESTED** [review]
Looks good outside some styling issues I left a few comments.

**Review by Lewin-B — CHANGES_REQUESTED** [review]
You're failing all the checks run "npm run lint:fix".
Also the search results are still white and the z index is still off

**Review by Lewin-B — APPROVED** [review]
Great job on the changes Lewin. looks good to me

**Pull request merged**